### DISCUSSION TITLE
v2.0.0-beta.3 - Open to any feedback!

### DISCUSSION THREAD

**Discussion by blindaa121** [PaloAltoNetworks/docusaurus-openapi-docs#571] [Announcements]
First and foremost, thank you all for your continued support and we hope you all have been enjoying `docusaurus-openapi-docs` thus far! We have recently been working on the upcoming `v2.0.0` release and would love to hear any thoughts, comments, and/or concerns from the community.

As we reach the final stretch, we welcome you all to test out the [latest beta release](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/releases/tag/v2.0.0-beta.3) as this entails much of the bigger changes to the plugin in terms of support for Docusaurus `2.4.0`, UI/UX improvements, bug fixes, and new features. 🎉

**Comment by george-xing** [comment]
@blindaa121 thank you! excited for v2.0.0, when do you expect it will be released?
